numpy/linalg/tests/test_linalg.py

```diff
@@ -189,28 +192,43 @@ def apply_tag(tag, cases):
                array([2., 1., 3.], dtype=double)),
     LinalgCase("csingle_nsq_1",
                array(
-                   [[1. + 1j, 2. + 2j, 3. - 3j], [3. - 5j, 4. + 9j, 6. + 2j]], dtype=csingle),
+                    [[1. + 1j, 2. + 2j, 3. - 3j], [3. - 5j, 4. + 9j, 6. + 2j]],
+                    dtype=csingle,
+                ),
```

Comment:
```suggestion
                   [[1. + 1j, 2. + 2j, 3. - 3j], [3. - 5j, 4. + 9j, 6. + 2j]],
                   dtype=csingle,
               ),
```

same for the ones below